src/__tests__/CompareResults/RevisionSelect.test.tsx

```diff
@@ -188,69 +130,4 @@ describe('Revision select', () => {
       }),
     ).toStrictEqual([]);
   });
-
-  it('Should render results for a998c42399a8', async () => {
```

Comment:
I removed this one completely because this was essentially the same test and testing the same behavior as the previous test block.